calibrations/mbd/MBDTRKZ/prorun_mbdtrkz.sh

```
@@ -0,0 +1,124 @@
+    -*)    usage ;;
+    *)     [[ -n "$runnumber" ]] && usage; runnumber=$1; shift ;;
+  esac
+done
+
+[[ -z "$runnumber" ]] && usage
+
+subdir="${ORIG_DIR}/MBDTRKZ/${runnumber}/$(printf '%04d' ${trkseg})"
+echo $subdir
+mkdir -p "${subdir}"
+
+cd "${subdir}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)